.github/workflows/scalability.yml

```diff
@@ -1,28 +1,35 @@
-name: Scalability Workflow
-# This workflow is triggered on pushes to the repository.
+name: Scalability Workflow updated
```

Comment:
Had to rename it to have it run on the new trigger. I will revert this and the trigger before merging.